src/blocks/_nav.scss

```diff
@@ -0,0 +1,48 @@
+.nav {
+  flex-shrink: 0;
+
+  &__list {
+    display: flex;
+    flex-direction: row;
+    flex-shrink: 0;
```

Comment:
same here
```suggestion
```

Reply:
Completely agree - thats my redunt code after experementing with various conditions .
Aprreciate you mentioning.